### DISCUSSION TITLE
Attempting to use Docker LXC  script and get an error at the end

### DISCUSSION THREAD

**Discussion by notchy12000** [tteck/Proxmox#2265] [Q&A]
Hi. Hope someone will be able to assist. I am attempting to install Frigate on docker LXC.
I used the following script.

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/docker.sh)"`

After inputting my values for setup and near the end of the install I get an error about "line 5540". 
How do I get this working?

![Screenshot 2023-12-28 130128](https://github.com/tteck/Proxmox/assets/78877501/0fa4733b-e98d-4e1e-96cc-15b8e5e8fe70)

**Comment by tteck** [comment]
It's _probably_ your storage, try using local-lvm.
